static/js/portico/signup.js

```diff
@@ -197,3 +197,29 @@ $(function () {
         this.parentNode.submit();
     });
 });
+
+$('#organization_type_select').on('change', function () {
+    const selected_object = document.getElementById("organization_type_select");
+    const selected_value = selected_object.options[selected_object.selectedIndex].value;
+    if (selected_value === "Open-source") {
+        document.getElementById("offer_details").style.display = "block";
```

Comment:
We generally use jQuery for this sort of manipulation.  I also think it'd be wise to write this in a more data-driven way as that'd be a lot easier to understand and test; you can look at the `apps_events` implementation in `landing-page.js` for inspiration.